src/components/templates/HeroSection.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Container, Button, ListItem } from "../index";
+
+const HeroSection = () => {
+  return (
+    <section>
+      <Container>
+        <div className=" flex flex-col items-center gap-y-4 sm:flex-row sm:gap-x-20 md:justify-between">
+          <div className="relative flex max-h-[440px] max-w-md items-end justify-center overflow-visible rounded-lg bg-secondary sm:w-1/2">
+            <img
+              src="/images/firstImage.png"
+              alt="hero-section"
+              className="w-full sm:h-auto sm:w-auto"
+            />
+            <ListItem
+              icon="/images/heartSkull.png"
+              text="More than 10K Patients treated!"
+              styles="absolute bg-white flex gap-x-2 rounded-lg h-15 w-32 bottom-8 right-0 p-1 lg:w-32 lg:bottom-80 lg:right-80 xl:right-96"
```

Comment:
I think we don't need to use height, and width properties, if needed then use max-width instead of width and height. Also, try to play with left-0 instead of right-96.
See the tailwind docs to get an idea of how to use these classes in negative way.